### ISSUE TITLE
Unit inconsistency in turbulent_kinetic_energy_dissipation_rate function

### ISSUE THREAD

**Issue by Wessel99** [contrailcirrus/pycontrails#76]
**Description**

My understanding is that the turbulent kinetic energy dissipation rate (or epsilon) should be in the unit of [m^2 / s^3]. However, whitin the calculation of epsilon, the square of w'n [m/s] is multiplied with the square of (enhanced) shear [s^-1], resulting in [m^2/ s^4].  This seems inconsistent.

I see this both in code and in "A contrail cirrus prediction model" Schumann 2012. 

Am I missing something or should this be corrected?

**Details**

- Version: [bv0.45.0]
- Module: [wake_vortex.py]
- OS: [windows]

**Additional Notes**

See snippet below:

def turbulent_kinetic_energy_dissipation_rate(
    ds_dz: npt.NDArray[np.float_],
    shear_enhancement_factor: npt.NDArray[np.float_] | float = 1.0,
) -> npt.NDArray[np.float_]:
    """
    Calculate the turbulent kinetic energy dissipation rate (epsilon).

    The shear enhancement factor is used to account for any sub-grid scale turbulence.

    Parameters
    ----------
    ds_dz : npt.NDArray[np.float_]
        Difference in wind speed over dz in the atmosphere, [:math:`m s^{-1} / m`]
    shear_enhancement_factor : npt.NDArray[np.float_] | float
        Multiplication factor to enhance the wind shear

    Returns
    -------
    npt.NDArray[np.float_]
        turbulent kinetic energy dissipation rate

    Notes
    -----
    See eq. (37) in :cite:`schumannContrailCirrusPrediction2012`.

    References
    ----------
    - :cite:`schumannContrailCirrusPrediction2012`
    """
    return 0.5 * 0.1**2 * (ds_dz * shear_enhancement_factor) ** 2

**Comment by roger-teoh** [comment]
Hi Wessel, 

Thank you for identifying this bug. We have managed to replicate the unit inconsistency in this function. Upon consulting with Ulrich Schumann, we identified that Eq. (37) in the CoCiP paper (Schumann, 2012) contains a print error where the S should not be squared. The correct equation can be found in Eq. (13) of Schumann and Gerz (1995):

_Schumann, U. and Gerz, T., 1995. Turbulent mixing in stably stratified shear flows. Journal of Applied Meteorology and Climatology, 34(1), pp.33-48._

We have created a separate branch `bug-fix/wake-vortex` to address this issue and strengthen the docstrings to include the units for each input and output parameters. This will be merged into the main branch, hopefully in the next couple of days. 

Regards,
Roger
